pkgs/development/python-modules/numexpr/default.nix

```diff
@@ -47,4 +41,4 @@ buildPythonPackage rec {
     homepage = "https://github.com/pydata/numexpr";
     license = lib.licenses.mit;
   };
-}
```

Comment:
The setup hook makes `mkl` and `numpy` "just work", without any customization needed in `numexpr`; this passes all of its test cases.